plugins/services/src/js/components/forms/NetworkingFormSection.tsx

```diff
@@ -59,7 +59,6 @@ class NetworkingFormSection extends mixin(StoreMixin) {
     super.componentDidMount?.();
     this.$dcosVersion = dcosVersion$.subscribe(({ hasCalicoNetworking }) => {
       this.setState({ hasCalicoNetworking });
-      console.log("hasCal", hasCalicoNetworking);
```

Comment:
i just had to remove that. ideally this would go on an own PR. please let me know how you feel about changes like this in the context of a feature-PR. @janisz